h2o-py/h2o/h2o.py

```diff
@@ -1409,6 +1409,18 @@ def download_model(model, path=""):
     path = os.path.join(os.getcwd() if path == "" else path, model.model_id)
     return api("GET /3/Models.fetch.bin/%s" % model.model_id, save_to=path)
 
+def upload_model(path):
+    """
+    Upload a model from the provided local path to the H2O cluster.
```

Comment:
please add a reference to the corresponding download/save function

maybe we should stress this is for binary models (not MOJO) and make it even clearer that local path means where the python client is running (I remember confusion about the difference of save/download model)